layouts/partials/mike-wolf-memorial-workdays-2025.html

```diff
@@ -0,0 +1,36 @@
+<div class="mike-wolf-memorial mt3">
+  <div class="flex">
+    <h2>Mike Wolf Memorial Defensible Workdays will be held on Saturday, June 28 and 29, 2025!</h2>
+  </div>
+
+  <h3>Don't Forget About the FREE Dumpsters!</h3>
```

Comment:
Do we want to provide a link here to explain what those are, or where to get them?

Reply:
Oh, I think I misunderstood: this *is* the notification about the free dumpsters and their locations.